twitter.py

```diff
@@ -196,7 +204,7 @@ def count_tweets(user: dict, from_date: datetime, to_date: datetime):
         from_date = from_date + timedelta(days=1)
 
     tweets_info = {
-        'info_date_tweet': date_tweet,
+        'info_date_when': date_tweet,
```

Comment:
renamed so it's called the same everywhere (no need to have 3x the same thing in the final dict)